lib/utils.js

```diff
@@ -2,6 +2,7 @@
 
 var _ = require('lodash');
 const MASK = 'XXXXX';
+var ALL_BODY = '*';
```

Comment:
Rename `ALL_BODY` to something more generic, as it's now relevant also for headers